python/lsst/ip/diffim/modelPsfMatch.py

```diff
@@ -372,15 +394,42 @@ def _buildCellSet(self, exposure, referencePsfModel):
                 widthList.append(widthS)
                 heightList.append(heightS)
 
-        lenMax = max(max(heightList), max(widthList))
-        lenMin = min(min(heightList), min(widthList))
+        psfSize = max(max(heightList), max(widthList))
 
-        lenPsfScience = lenMax if self.config.padPsf else lenMin
-        dimenS = afwGeom.Extent2I(lenPsfScience, lenPsfScience)
+        if self.config.doAutoPadPsf:
+            minPsfSize = nextOddInteger(self.kConfig.kernelSize*self.config.autoPadPsfTo)
```

Comment:
The speed difference here comes from setupBitFlip using int(np.ceil( and setupIfMod not, it seems to just be doing the mod on whatever type that x is. Over all I don't have a strong feeling on this and reuse in the unit test is a good enough argument for me.